src/components/modals/Modal.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ReactModal from 'react-modal';
+import classNames from 'classnames';
+
+import FAIcon from 'components/FAIcon';
+
+const CONTENT_CLASS_NAME = 'react-modal__content';
+
+const Modal = ({ isOpen=false, title='', closeModal, children, contentModifiers=[] }) => {
+    const modifiedClassNames = contentModifiers.map(modifier => `${CONTENT_CLASS_NAME}--${modifier}`);
```

Comment:
don't we have a `getBEMClassName` in utils somewhere?

Reply:
Yes but won't this add `open-forms` to the start of the className, which we don't want since we're overriding the styles coming from React Modal?

Reply:
Nevermind, I realise now that react modal does not come with it's own styling.  I'll have this updated in a sec.